README.md

```diff
@@ -13,143 +15,6 @@
-| redis         | default | (none)                 | `host`, `port`, `username`, `password`                                                                                                                          |
-| postgres      | default | (none)                 | `host`, `port`, `name` (aka `database`), `username`, `password`                                                                                                 |
-| mysql         | default | (none)                 | `host`, `port`, `name` (aka `database`), `username`, `password`                                                                                                 |
-| s3            | default | (none)                 | `endpoint`, `access_key_id`, `secret_key`, `bucket`, with `region=""`, `aws_access_key_id=<access_key_id>`, and `aws_secret_key=<secret_key>` for compatibility |
-| dns           | default | (none)                 | `host`                                                                                                                                                          |
-| route         | default | `host`, `path`, `port` |                                                                                                                                                                 |
-| amqp          | default | (none)                 | `host`, `port`, `vhost`, `username`, `password`                                                                                                                 |
-| mongodb       | default | (none)                 | `host`, `port`, `username`, `password`, `connection`                                                                                                            |
-| kafka-topic   | default | (none)                 | `host`, `port`, `name`, `num_partitions`                                                                                                                        |
-| elasticsearch | default | (none)                 | `host`, `port`, `username`, `password`                                                                                                                          |
-
-These can be found in the default provisioners file. You are encouraged to write your own provisioners and add them to the `.score-compose` directory (with the `.provisioners.yaml` extension) or contribute them upstream to the [default.provisioners.yaml](internal/command/default.provisioners.yaml) file.
-
-## Installation
-
-To install `score-compose`, follow the instructions as described in our [installation guide](https://docs.score.dev/docs/score-implementation/score-compose/#installation). You will also need a recent version of Docker and the Compose plugin installed. Read more [here](https://docs.docker.com/compose/install/).
```

Comment:
Why remove this? it's already linking to the docs and makes it very easy for someone scanning the repo to see.